lib/helper/Nightmare.js

```diff
@@ -232,6 +245,22 @@ class Nightmare extends Helper {
     }
   }
 
+  /**
+   * Get HAR
+   *
+   * ```js
+   * let har = yield I.grabHAR();
+   * fs.writeFileSync('sample.har', JSON.stringify({log: har}));
+   * ```
+   */
+  async grabHAR() {
```

Comment:
can you make a function to saveHAR as well? 
Looks like that would be useful too

Reply:
added saveHar()